ci/pipeline.yml

```diff
@@ -128,11 +128,6 @@ resources:
       repository: spring-native
```

Comment:
why does the PR modify our ci pipeline?

Reply:
no earthly idea. please feel free to ignore any changes to the CI pipeline